A32NX/html_ui/Pages/VCockpit/Instruments/Airliners/A320_Neo/EICAS/ECAM/A320_Neo_LowerECAM_CRZ.js

```diff
@@ -0,0 +1,122 @@
+var A320_Neo_LowerECAM_CRZ;
+(function (A320_Neo_LowerECAM_CRZ) {
+    class Definitions {
+    }
+    A320_Neo_LowerECAM_CRZ.Definitions = Definitions;
+    class Page extends Airliners.EICASTemplateElement {
+        constructor() {
+            super();
+            this.isInitialised = false;
+        }
+        get templateID() { return "LowerECAMCRZTemplate"; }
+        connectedCallback() {
+            super.connectedCallback();
+            TemplateElement.call(this, this.init.bind(this));
+        }
+        init() {
+            this.FuelUsedTotal = this.querySelector("#FuelUsedTotal"); //
+            this.FuelUsedLeft = this.querySelector("#FuelUsedLeft"); //
+            this.FuelUsedRight = this.querySelector("#FuelUsedRight"); //
+
+            this.OilQuantityLeft = this.querySelector("#OilQuantityLeft"); //
+            this.OilQuantityLeftDecimal = this.querySelector("#OilQuantityLeftDecimal");
+            this.OilQuantityRight = this.querySelector("#OilQuantityRight"); //
+            this.OilQuantityRightDecimal = this.querySelector("#OilQuantityRightDecimal");
+
+            this.VibN1Left = this.querySelector("#VibN1Left"); //
+            this.VibN1LeftDecimal = this.querySelector("#VibN1LeftDecimal");
+            this.VibN1Right = this.querySelector("#VibN1Right"); //
+            this.VibN1RightDecimal = this.querySelector("#VibN1RightDecimal");
+
+            this.VibN2Left = this.querySelector("#VibN2Left"); //
+            this.VibN2LeftDecimal = this.querySelector("#VibN2LeftDecimal");
+            this.VibN2Right = this.querySelector("#VibN2Right"); //
+            this.VibN2RightDecimal = this.querySelector("#VibN2RightDecimal");
+
+            this.LandingElevationMode = this.querySelector("#LandingElevationMode");
+            this.LandingElevation = this.querySelector("#LandingElevation");
+
+            this.DeltaPressure = this.querySelector("#DeltaPressure");
+            this.DeltaPressureDecimal = this.querySelector("#DeltaPressureDecimal");
+
+            this.CabinVerticalSpeed = this.querySelector("#CabinVerticalSpeed"); //
+            this.CabinAltitude = this.querySelector("#CabinAltitude"); //
```

Comment:
Why the `//` at the end of those lines ?

Reply:
I used them to remember which of those I already integrated in the JS, forgot to remove afterwards.